dcos/package.py

```diff
@@ -1321,6 +1328,16 @@ def has_command_definition(self, revision):
 
         return self.has_definition(revision, 'command.json')
 
+    def has_resource_definition(self, revision):
```

Comment:
We added a few new methods to this file. Can we make them private if they are not used outside of this class? It looked like they are all used in the `marathon_json` method.